### ISSUE TITLE
zos-ssh command returns even the prompt and issuing command in stdout

### ISSUE THREAD

**Issue by weinfurt** [zowe/zowe-cli#1724]
**Describe the bug**
When issuing zos ssh commands I always get the prompt character and the issuing command in the stdout. Even the `--response-format-json` flag does not help. Sometimes, when the command is long it gets trimmed. Here are some examples:

![image](https://github.com/zowe/zowe-cli/assets/10806904/c28e8a94-e738-453a-8a44-b363a1cf7f00)

Here is the output even more corrupted:
![image](https://github.com/zowe/zowe-cli/assets/10806904/7a3de9b6-502f-4be4-b9d3-2b87a63cfc39)

**Expected and actual results**
only the command output should be returned and no prompt. Like running the command on the system itself.
![image](https://github.com/zowe/zowe-cli/assets/10806904/80df6bb4-a33d-44c7-9978-5de78fe7bd47)

**Describe your environment**
Zowe v7.16.4
Node v19.9.0
Npm v9.6.3
Ubuntu bash in WSL

**Additional context**

**Comment by weinfurt** [comment]
I spent some time to investigate it and maybe workaround this and I figured out that the output is dependent on some setup in my `.profile` config file in my home directory. I am using `bash` by default so this is the last command in my `.profile` file. If I remove the `bash` command and also the configuration of my default text editor (vi), then I will get output with just the default `$` prompt and then the expected command output. It is not perfect but at least I am not getting the clutter from the command line itself.
This is what I am getting with disabled bash and default editor:
![image](https://github.com/zowe/zowe-cli/assets/10806904/458fd56a-1312-4bad-8ade-fb147f1fdbf0)

The only extraneous character in the output is the `$` prompt sign and some white spaces

I can share my .profile file if needed.
